src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,109 @@
+import cN from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+import { Loader } from '../Loader';
```

Comment:
Ensure that the `Loader` component is correctly implemented and imported. If it's not used elsewhere, consider removing the commented import in `App.tsx`.